### PR TITLE
📝 Document new `oauthLogin`

### PR DESCRIPTION
**Pull request by coyotte508**
Last PR for a while in the docs regarding OAuth I think! (unless we add new scopes :) )

cc @jbilcke-hf @xenova for viz

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Looks good! :tada:

File: docs/hub/spaces-oauth.md [hunk 1]
```diff
@@ -96,3 +98,4 @@ You should use `target=_blank` on the button to open the sign-in page in a new t
 - [Gradio test app](https://huggingface.co/spaces/Wauplin/gradio-oauth-test)
 - [Hugging Chat (NodeJS/SvelteKit)](https://huggingface.co/spaces/huggingchat/chat-ui)
 - [Inference Widgets (Auth.js/SvelteKit)](https://huggingface.co/spaces/huggingfacejs/inference-widgets), uses the `inference-api` scope to make inference requests on behalf of the user.
+- [Client-Side in a Static Space (huggingface.js)](https://huggingface.co/spaces/huggingfacejs/client-side-oauth) - very simple JavaScript example.
```
Comment (julien-c) [inline docs/hub/spaces-oauth.md:101]: might be worth copy/pasting a super simple snippet here in the doc?

**Review by julien-c — APPROVED** [review]
looks good, now let's make usage big(er) 🔥

File: docs/hub/spaces-oauth.md [hunk 2]
```diff
@@ -77,7 +77,9 @@ Those scopes are optional and can be added by setting `hf_oauth_scopes` in your
 
 ## Adding the button to your Space
 
-You now have all the information to add a "Sign-in with HF" button to your Space. Some libraries ([Python](https://github.com/lepture/authlib), [NodeJS](https://github.com/panva/node-openid-client)) can help you implement the OpenID/OAuth protocol. Gradio also provides **built-in support**, making implementing the Sign-in with HF button a breeze; you can [check out the associated guide](https://www.gradio.app/guides/sharing-your-app#o-auth-login-via-hugging-face).
+You now have all the information to add a "Sign-in with HF" button to your Space. Some libraries ([Python](https://github.com/lepture/authlib), [NodeJS](https://github.com/panva/node-openid-client)) can help you implement the OpenID/OAuth protocol. 
+
+Gradio and hugginface.js also provides **built-in support**, making implementing the Sign-in with HF button a breeze; you can check out the associated guides with [gradio](https://www.gradio.app/guides/sharing-your-app#o-auth-login-via-hugging-face) and with [hugginface.js](https://huggingface.co/docs/huggingface.js/hub/README#oauth-login).
```
Comment (osanseviero) [inline docs/hub/spaces-oauth.md:82]:
```suggestion
Gradio and hugginface.js also provide **built-in support**, making implementing the Sign-in with HF button a breeze; you can check out the associated guides with [gradio](https://www.gradio.app/guides/sharing-your-app#o-auth-login-via-hugging-face) and with [hugginface.js](https://huggingface.co/docs/huggingface.js/hub/README#oauth-login). 
```

**Review by osanseviero — APPROVED** [review]
Nice! :fire:

**Pull request merged**